system/jlib/jsocket.cpp

```diff
@@ -4853,7 +4866,7 @@ class CSocketEpollThread: public CSocketBaseThread
         return false;
     }
 
-    bool remove(ISocket *sock)
+    bool remove(ISocket *sock, unsigned numThreads)
     {
         if (terminating)
```

Comment:
Does this need to be inside the critical block?

Reply:
a good idea, I have moved it